roles/coreos_bootstrap/tasks/main.yml

```diff
@@ -8,3 +8,18 @@
 - name: Run bootstrap.sh
   script: bootstrap.sh
   when: need_bootstrap | failed
+
+- name: Setup timesyncd.conf
+  become: yes
+  template:
+    src: "{{ role_path }}/templates/timesyncd.conf.j2"
+    dest: /etc/systemd/timesyncd.conf
+    owner: root
+    group: root
+    mode: 0644
+  notify:
+    - restart systemd-timesyncd
+
+- name: Setup timezone
+  become: yes
+  command: timedatectl set-timezone {{ timezone }}
```

Comment:
Dette steget tryner på linting, så enten må vi ha en sjekk eller så må vi skippe linting, 
Jeg tenker vel at for dette så er det greit å skippe linting,
I.e  tags:
      -  skip_ansible_lint

Reply:
Kan vi bruke en modul?
https://docs.ansible.com/ansible/2.6/modules/timezone_module.html

Reply:
Ah, en modul for det også. Jeg forsøker med den. Bedre det.